app/src/main/java/amazingme/activities/app/ChildRegistrationActivity.java

```diff
@@ -43,29 +43,30 @@ public void bindToUserInterface() {
         //backBtn.setOnClickListener(new View.OnClickListener() {
         //@Override
         //public void onClick(View v) {
-        backBtn = (Button) findViewById(user_profile_back_button);
-        doneBtn = (Button) findViewById(R.id.button3);
-        childBtn = (Button) findViewById(R.id.button2);
-        firstnameEditText = (EditText) findViewById(R.id.editText2);
-        lastnameEditText = (EditText) findViewById(R.id.editText4);
-        ageEditText = (EditText) findViewById(R.id.editText5);
+        profilepic = (QuickContactBadge) findViewById(R.id.child_registration_child_portrait);
+        backBtn = (Button) findViewById(R.id.child_registration_back_button);
+        doneBtn = (Button) findViewById(R.id.child_registration_done_button);
+        childBtn = (Button) findViewById(R.id.child_registration_add_another_child);
+        firstnameEditText = (EditText) findViewById(R.id.child_registration_first_name);
+        lastnameEditText = (EditText) findViewById(R.id.child_registration_last_name);
+        ageEditText = (EditText) findViewById(R.id.child_registration_date_of_birth);
         //sexEditText = (EditText) findViewById(R.id.editText7);
-        profview = (TextView) findViewById(R.id.textView4);
-        abilityview = (TextView) findViewById(R.id.textView5);
-        sexview = (TextView) findViewById(R.id.lable);
+        profview = (TextView) findViewById(R.id.user_profile_parent_profile_label);
+        abilityview = (TextView) findViewById(R.id.child_registration_question_label);
+        sexview = (TextView) findViewById(R.id.child_registration_sex_label);
 
-        yesnoSpinner = (Spinner) findViewById(R.id.spinner1);
-        malefemale = (Spinner) findViewById(R.id.spinner2);
+        yesnoSpinner = (Spinner) findViewById(R.id.child_registration_disability_dropdown);
+        malefemale = (Spinner) findViewById(R.id.child_registration_sex_dropdown);
```

Comment:
let's just call this the sexSpinner. it sounds goofy but whatever it's what it is.